src/PrettierEditProvider.ts

```diff
@@ -57,16 +77,24 @@ function format(text: string, path: string): string {
         semi: config.semi,
         useTabs: config.useTabs,
     };
+
     if (config.eslintIntegration) {
+        if (isNonJsParser) {
+            // not supported yet
+            return text;
+        }
         const prettierEslint = require('prettier-eslint') as PrettierEslintFormat;
         return prettierEslint({
             text,
-            filePath: path,
-            fallbackPrettierOptions: prettierOptions
+            filePath: fileName,
+            fallbackPrettierOptions: prettierOptions,
         });
     }
-
-    const prettier = requireLocalPkg(path, 'prettier') as Prettier;
+    const prettier = requireLocalPkg(fileName, 'prettier') as Prettier;
+    if (isNonJsParser && semver.lt(prettier.version, '1.4.0-beta')) {
+        // not supported yet
+        return text;
```

Comment:
I'm not sure silently doing *nothing* is the best option here.
I would throw or silently format with embed version.
Don't know which of these would be better.

Or maybe format with embed and show a warning.